cincinnati/src/lib.rs

```diff
@@ -21,6 +21,9 @@ extern crate serde_derive;
 #[macro_use]
 pub mod plugins;
 
+#[macro_use]
+extern crate trackable;
```

Comment:
This throws a warning here, since crate macros are being used later but not directly here. Is there a better way to import it?

Reply:
You should be able to place `use trackable::macro_name` in the scope where you need the macro.